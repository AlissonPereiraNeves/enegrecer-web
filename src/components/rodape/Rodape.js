import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import logo from '../../imagens/logo-rodape.svg';
import facebook from '../../imagens/facebook.svg';
import instagram from '../../imagens/instagram.svg';
import './rodape.css';

export default class Rodape extends Component {
  componentWillMount() {
  }
  
  render() {
    return (
      <div className="rodape">
        <div>
          <img src={logo} alt="Logo Verdade Seja Dita" />          
        </div>
        <div className="direitos">
          <label>Â© 2018 - Verdade Seja Dita todos os direitos reservados</label>
        </div>
        <div className="redes-sociais">
          <div className="facebook">
            <a href="https://pt-br.facebook.com/Enegrecer/" target="_blank">
              <img src={facebook} alt="Logo Facebook" />
            </a>
          </div>
          <div className="instagram">
            <a href="https://www.instagram.com/coletivoenegrecer/" target="_blank">
              <img src={instagram} alt="Logo Instagram" />
            </a>
          </div>
        </div>        
      </div>  
    );
  }
}
