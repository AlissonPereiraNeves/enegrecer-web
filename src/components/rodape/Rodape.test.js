import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Rodape from './Rodape';

describe('Rodape', () => {
  it('deve construir a pagina corretamente ', () => {
    const rodape = shallow(<Rodape />);
    const rodapeJson = toJson(rodape);
    expect(rodapeJson).toMatchSnapshot();
  });
});
