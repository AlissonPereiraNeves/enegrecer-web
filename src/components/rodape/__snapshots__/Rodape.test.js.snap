// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rodape deve construir a pagina corretamente  1`] = `
<div
  className="col s12 m6 rodape"
>
  <div>
    <img
      alt="Logo Verdade Seja Dita"
      src="logo-rodape.svg"
    />
  </div>
  <div
    className="direitos"
  >
    <label
      htmlFor="direitos"
    >
      Â© 2018 - Verdade Seja Dita todos os direitos reservados
    </label>
  </div>
  <div
    className="redes-sociais"
  >
    <div
      className="facebook"
    >
      <a
        href="https://pt-br.facebook.com/Enegrecer/"
        target="blank"
      >
        <img
          alt="Logo Facebook"
          src="facebook.svg"
        />
      </a>
    </div>
    <div
      className="instagram"
    >
      <a
        href="https://www.instagram.com/coletivoenegrecer/"
        target="blank"
      >
        <img
          alt="Logo Instagram"
          src="instagram.svg"
        />
      </a>
    </div>
  </div>
</div>
`;
